narwhal/primary/src/core.rs

```diff
@@ -413,6 +422,8 @@ impl Core {
         }?;
 
         // Broadcast the certificate.
+        debug!("Broadcast certificate {:?}", certificate);
```

Comment:
There are already a couple other log messages that at least partially overlap with this:
- line 378 (which should happen right before broadcast unless Core loop is getting gunked up with other stuff in between)
- anemo itself logs the SendMessage RPCs, which are now only used for broadcasting certs as all other primary messages have been removed

In light of that I'm wondering if adding this one too would be too spammy? But if you added this based on your own experience with reading the logs and wishing this were here, then I guess better a bit too much than a bit too little!

Reply:
yeah, you are right. And we also have a debug level instrument covering the function. Removing.